packages/ckeditor5-link/src/linkconfig.ts

```diff
@@ -9,6 +9,44 @@
 
 import type { ArrayOrItem } from 'ckeditor5/src/utils.js';
 
+/**
+ * This is a list of most commonly used protocols to make the link custom configuration a bit easier.
+ * The {@link module:link/linkconfig~LinkConfig#allowedProtocols `allowedProtocols` } list
+ * can also contain other values than enlisted here.
+ */
+export enum COMMON_PROTOCOLS {
+	HTTPS = 'https?',
+	FTPS = 'ftps?',
+	HTTPS_ONLY = 'https',
+	SFTP = 'sftp',
+	TEL = 'tel',
+	SMS = 'sms',
+	MAIL = 'mailto',
+	SSH = 'ssh',
+	GIT = 'git',
+	SVN = 'svn',
+	SAMBA = 'smb',
+	XMPP = 'xmpp',
+	SKYPE = 'skype',
+	SLACK = 'slack',
+	VIBER = 'viber',
+	JABBER = 'jabber',
+	ZOOMMTG = 'zoommtg',
+	ZOOMUS = 'zoomus'
+}
+
+/**
+ * @internal
+ */
+export const DEFAULT_LINK_PROTOCOLS = [
```

Comment:
First of all, I'm not sure what type of breaking changes when you extend the list of protocols, not omitting any of the old ones.
Second, well, I did this because this is how I interpreted the connected tickets, for instance:
https://github.com/ckeditor/ckeditor5/issues/11855

but probably they will simply need to make custom configs then. fine both ways.

Reply:
added were sftp, tel and sms, as I took it from the pinned related issues.

Reply:
Just to clarify, changing default behaviors, whether we're limiting or expanding functionality, can lead to breaking changes. For example, if someone doesn't want to support a specific protocol (which was not previously supported by default), they'll have to modify their code. It might not be a big deal, but it still provides a minor breaking change. Remember that different clients/users might have different situations or requirements in their applications.

Secondly, as Szymon mentioned, that was not the point of this task.

Reply:
Okay then, just to be completely clear. I thought it **is** a scope of this task just because of how the linked issues about several protocols (which were clearly request to add support for them) were described and concluded. But it's also true that in the main #14304 we are focused only on adding of the customization option. So will keep the original list countet to my prior intuition.

Reply:
@Witoso Could you take a look? I think idea of adding or not adding any of the requested link protocols to the default set deserves one more opinion, if you will.